src/pages/vrf/v2/subscription/migration-from-v1.mdx

```diff
@@ -29,15 +29,18 @@ Read the [Chainlink VRF v2 blog post](https://blog.chain.link/vrf-v2-mainnet-lau
 
 ## Updating your applications to use VRF v2
 
-<VrfCommon callout="ui"/>
+<VrfCommon callout="ui" />
 
 To modify your existing smart contract code to work with VRF v2, complete the following changes. See the [Get a Random Number](/vrf/v2/subscription/examples/get-a-random-number/) guide for an example.
 
 1. Set up and fund a subscription in the Subscription Manager at [vrf.chain.link](https://vrf.chain.link).
-   <!-- prettier-ignore -->
+   {/* prettier-ignore */}
+
    <div class="remix-callout">
```

Comment:
This needs the CSS fix. https://documentation-git-fork-thedriftofwords-mdx-vrf-chainlinklabs.vercel.app/vrf/v2/subscription/migration-from-v1#updating-your-applications-to-use-vrf-v2

Reply:
Thanks. For whatever reason, removing target="_blank" seems to fix it.